packages/template-retail-react-app/app/ssr.js

```diff
@@ -59,6 +61,34 @@ const {handler} = runtime.createHandler(options, (app) => {
         })
     )
 
+    const createSlasHandler = ({clientId, secret, shortCode}) => {
```

Comment:
The main event happens here. A SLAS request coming from `commerce-sdk-react` is intercepted and we add the client secret as an auth header to `/token` calls.

We may want to allow developers to set additional endpoints where this client secret is added (ie. `/password/action` for password reset).